coral/src/app/components/ComplexNativeSelect.tsx

```diff
@@ -0,0 +1,73 @@
+import { NativeSelect } from "@aivenio/aquarium";
+import { ChangeEvent } from "react";
+
+type ComplexNativeSelectProps<T> = {
+  options: Array<T>;
+  onBlur: (option: T | undefined) => void;
+  optionToString: (opt: T) => string;
+  getValue: (opt: T) => string;
+  placeholder: string;
+  labelText: string;
+  disabled?: boolean;
+  defaultValue?: string;
+  required?: boolean;
+  error?: string;
+};
+
+function ComplexNativeSelect<T>(props: ComplexNativeSelectProps<T>) {
+  const {
+    options,
+    onBlur,
+    disabled = false,
+    required = false,
+    defaultValue,
+    labelText,
+    optionToString,
+    placeholder,
+    getValue,
+    error,
+  } = props;
+
+  // the placeholder behavior will be covered by DS NativeSelect
+  // soon, this is a temp solution
+  const placeholderValue = "055d87f2-8cd4-11ed-a1eb-0242ac120002";
```

Comment:
Lets extract this outside of the component. It is a true constant.

Reply:
This is only temporary until (probably) the next DS release when `NativeSelect` is going to cover it. Do you think it's worth moving it in that case?